sidebarCommunity.js

```diff
@@ -0,0 +1,16 @@
+
+
+module.exports = {
```

Comment:
this file should not be there, since its in the language specific folders already.

Put it inside the `.gitignore` file like this: `/sidebarCommunity.js` next to the normal sidebar.js